step.sh

```diff
@@ -33,5 +33,5 @@ else
   # Export the bad json into a file
   curl --silent "$collector_url/micro/bad" | jq '.' >> $BITRISE_DEPLOY_DIR/snowplow_bad.json
 
-  exit 1
+  # exit 1 # Since Remote Snowplow Micro is not stable, we are not going to failed the build when there is bad event. Will comeback to it once it is stable
```

Comment:
I am not sure if we need to avoid exit when total event is 0 😂 , although it will not happened that often 😂 (Unless server down)